docs/tutorials/using_apis/Mbed_USB_WAV_Audio_Player/Mbed_USB_WAV_Audio_Player.md

```diff
@@ -1,6 +1,4 @@
-# Mbed USB WAV audio player
-
-<!--since the library isn't called Mbed USB, just USB, and since this lives in the Mbed docs - can we remove "mbed" from the title?-->
+# USB WAV audio player
 
 This tutorial explains how to put together a USB WAV audio player with Mbed OS. You will load a WAV file onto an SD card connected to an Mbed board, and then connect that board to your computer. The Mbed board acts as the computer's USB microphone. The sound that the microphone is "recording" is the data on the SD card that it sends to the host computer. This allows the sound to be played on speakers or headphones or recorded with a program, such as Audacity.
 
```

Comment:
Sorry, I meant to comment earlier, but I do actually think some detail is lost by changing "You can then play the WAV file from the board on a host computer over USB" to "and then connect that board to your computer"